### ISSUE TITLE
Unknown units error for units: `FEET`

### ISSUE THREAD

**Issue by EvanBianco** [kinverarity1/lasio#164]
Attempting to read a LAS file with `STRT`, `STOP` and `DEPT` units specified in `.FEET`. 

![image](https://user-images.githubusercontent.com/1699959/31894543-2ce273f6-b7e5-11e7-9534-edb5f594550b.png)

Shouldn't break and should handle this nicely.

```
/Applications/anaconda/envs/caml/lib/python3.4/site-packages/lasio-0.11.2-py3.4.egg/lasio/las.py in depth_m(self)
    317         else:
    318             raise exceptions.LASUnknownUnitError(
--> 319                 'Unit of depth index not known')
    320 
    321     @property

LASUnknownUnitError: Unit of depth index not known
```

**Comment by kinverarity1** [comment]
Thanks for bringing this up - I've been meaning to fix it for a while! (#74)

**Comment by kinverarity1** [comment]
OK should be all good, I've added @kwinkunks' changes in  6245a10 (now in master) and a unit test similar to your example above.
